src/components/ModelTableList/_model-table-list.scss

```diff
@@ -70,4 +51,83 @@
       }
     }
   }
+
+  .status-group {
+    td,
+    th {
+      @include mobile-and-medium {
+        // Credential
+        &:nth-child(5),
+        // Controller
+        &:nth-child(6) {
+          display: none;
+        }
+      }
+
+      @include mobile {
+        // Summary icons
+        &:nth-child(2) {
+          display: none;
+        }
+        // Last updated
+        &:nth-child(7) {
+          display: none;
+        }
```

Comment:
In all 3 places for `@include mobile`, was wondering if maybe we can format it the same as `@include mobile-and-medium` for consistency reason?

```
        // Summary icons
        &:nth-child(2),
        // Last updated
        &:nth-child(7) {
          display: none;
        }
```

Reply:
Done!